rust/agents/relayer/src/settings/mod.rs

```diff
@@ -11,6 +11,8 @@ decl_settings!(Relayer {
     maxprocessingretries: String,
     /// The multisig checkpoint syncer configuration
     multisigcheckpointsyncer: abacus_base::MultisigCheckpointSyncerConf,
-    /// This is optional. See `Whitelist` for more.
+    /// This is optional. See `MatchingList` for more.
```

Comment:
nit: it'd be nice to make this mention something about what the whitelist/blacklist is and/or what passing `None` means